plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -294,6 +298,9 @@
     private static final String ORC_BLOOM_FILTER_COLUMNS_KEY = "orc.bloom.filter.columns";
     private static final String ORC_BLOOM_FILTER_FPP_KEY = "orc.bloom.filter.fpp";
 
+    private static final String PARQUET_BLOOM_FILTER_COLUMNS_KEY = "parquet.bloom.filter.columns";
+    private static final String PARQUET_BLOOM_FILTER_FPP_KEY = "parquet.bloom.filter.fpp";
```

Comment:
Is this a well-known property (i.e., understood by other systems like Hive or Spark) or are we just making it up?

Reply:
Good point, Hive doesn't expose these as property keys: https://github.com/apache/parquet-mr/blob/master/parquet-hadoop/README.md

Reply:
@martint On a separate note, do you know if there's a better way to access and provide the Iceberg table properties (containing bloom filter fpp and columns) to `IcebergFileWtiter#createParquetWriter`?

Reply:
Let me look into that. I haven't reviewed that part of the code yet.